CMakeLists.txt

```diff
@@ -262,6 +262,7 @@ set(sensor_msgs
   msgs/Range.proto
   msgs/SITLGps.proto
   msgs/OpticalFlow.proto
+  msgs/Barometer.proto
```

Comment:
Barometer is the sensor. The data should be named depending on the type of data being propagated. In this case, something like `Pressure` (similar to ROS `FluidPressure`) makes more sense.

Reply:
Agree, it's done.